ledger/participant-integration-api/src/main/scala/platform/apiserver/services/tracking/TrackerImpl.scala

```diff
@@ -76,9 +78,12 @@ private[services] object TrackerImpl {
         Ctx[Promise[Completion], Completion],
         Materialized[NotUsed, Promise[Completion]]],
       inputBufferSize: Int,
+      sizeCounter: Counter,
+      saturationCounter: Counter,
   )(implicit materializer: Materializer, loggingContext: LoggingContext): TrackerImpl = {
-    val ((queue, mat), foreachMat) = Source
-      .queue[QueueInput](inputBufferSize, OverflowStrategy.dropNew)
+    sizeCounter.inc(inputBufferSize.toLong)
```

Comment:
This is never decremented and we keep incrementing it every time the tracker gets created. I've refactored the code to handle the inc/dec in InstrumentedSource instead.